include/ceed/ceed.h

```diff
@@ -745,6 +746,7 @@ CEED_EXTERN int CeedOperatorGetCeed(CeedOperator op, Ceed *ceed);
 CEED_EXTERN int CeedOperatorGetNumElements(CeedOperator op, CeedInt *num_elem);
 CEED_EXTERN int CeedOperatorGetNumQuadraturePoints(CeedOperator op,
     CeedInt *num_qpts);
+CEED_EXTERN int CeedOperatorGetFlopsEstimate(CeedOperator op, CeedInt *flops);
```

Comment:
This could overflow long before the vector lengths overflow.
```suggestion
CEED_EXTERN int CeedOperatorGetFlopsEstimate(CeedOperator op, CeedSize *flops);
```